### ISSUE TITLE
Cannot open launcher on MacOS

### ISSUE THREAD

**Issue by chrisguida** [lightning-power-users/node-launcher#308]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] support request

* **What is the current behavior?**
Crashing on launch

* **If the current behavior is a bug, please provide the steps to reproduce**
Download launcher on Mac 10.14.3, open. Immediately get an error.

* **What is the expected behavior?**
Launcher should open

* **What is the motivation / use case for changing the behavior?**

* **Please tell us about your environment:**

  - Operating System: macOS

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

File "node_launcher/gui/system_tray_widgets/lnd_output_widget.py", line 147, in handle_unlock_wallet File "node_launcher/gui/system_tray_widgets/lnd_output_widget.py", line 96, in generate_seed File "node_launcher/node_set/lnd_client/lnd_client.py", line 144, in generate_seed File "grpc/_channel.py", line 549, in __call__ File "grpc/_channel.py", line 466, in _end_unary_response_blocking

**Comment by PierreRochard** [comment]
Yikes, what does your LND log say?

**Comment by chrisguida** [comment]
How do I find the LND log? This is my first time trying the launcher, just following the instructions here: https://medium.com/lightning-power-users/windows-macos-lightning-network-284bd5034340

Am I supposed to download and install lnd separately?

Thanks :)

**Comment by PierreRochard** [comment]
Nope, Node Launcher gets lnd for you, go to ~/Library/Application Support/lnd/logs/bitcoin/mainnet/lnd.log

**Comment by chrisguida** [comment]
Looks like that file doesn't exist. Actually the lnd directory underneath
Application Support isn't there either.

**Comment by sr1919** [comment]
I faced the same issue on High Sierra (v10.13.6), NL v6.0.4
Here is log:

2019-04-17 17:04:50.164 [INF] LTND: Version: 0.6.0-beta commit=v0.6-beta-rc4, build=production, logging=default
2019-04-17 17:04:50.164 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2019-04-17 17:04:50.173 [INF] CHDB: Checking for schema update: latest_version=8, db_version=8
2019-04-17 17:04:50.187 [INF] RPCS: password RPC server listening on 127.0.0.1:10009
2019-04-17 17:04:50.199 [INF] RPCS: password gRPC proxy started at 127.0.0.1:8080
2019-04-17 17:04:50.200 [INF] LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.

**Comment by chrisguida** [comment]
Oh, I'm dumb, looks like the path is actually /Users/chris/Library/Application Support/Lnd/logs/bitcoin/mainnet/lnd.log, with a capital "Lnd" rather than "lnd".

Anyway, here's the file's contents:

2019-04-16 12:19:13.942 [INF] LTND: Version: 0.6.0-beta commit=v0.6-beta-rc4, build=production, logging=default
2019-04-16 12:19:13.942 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2019-04-16 12:19:13.975 [INF] CHDB: Checking for schema update: latest_version=8, db_version=8
2019-04-16 12:19:13.977 [INF] RPCS: password RPC server listening on 127.0.0.1:10009
2019-04-16 12:19:13.977 [INF] RPCS: password gRPC proxy started at 127.0.0.1:8080
2019-04-16 12:19:13.978 [INF] LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.
2019-04-16 12:22:50.286 [INF] LTND: Version: 0.6.0-beta commit=v0.6-beta-rc4, build=production, logging=default
2019-04-16 12:22:50.286 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2019-04-16 12:22:50.287 [INF] CHDB: Checking for schema update: latest_version=8, db_version=8
2019-04-16 12:22:50.288 [INF] RPCS: password RPC server listening on 127.0.0.1:10009
2019-04-16 12:22:50.288 [INF] RPCS: password gRPC proxy started at 127.0.0.1:8080
2019-04-16 12:22:50.288 [INF] LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.

**Comment by tonic20** [comment]
I have same issue on MacOS Mojave 10.14.3 (18D109) 
NL v6.0.5
Here is log:

```
2019-04-25 13:07:43.574 [INF] LTND: Version: 0.6.0-beta commit=v0.6-beta, build=production, logging=default
2019-04-25 13:07:43.574 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2019-04-25 13:07:43.625 [INF] CHDB: Checking for schema update: latest_version=8, db_version=8
2019-04-25 13:07:43.631 [INF] RPCS: password RPC server listening on 127.0.0.1:10009
2019-04-25 13:07:43.631 [INF] RPCS: password gRPC proxy started at 127.0.0.1:8080
2019-04-25 13:07:43.631 [INF] LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.
2019-04-25 13:08:15.273 [INF] LTND: Version: 0.6.0-beta commit=v0.6-beta, build=production, logging=default
2019-04-25 13:08:15.273 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2019-04-25 13:08:15.274 [INF] CHDB: Checking for schema update: latest_version=8, db_version=8
2019-04-25 13:08:15.275 [INF] RPCS: password RPC server listening on 127.0.0.1:10009
2019-04-25 13:08:15.275 [INF] RPCS: password gRPC proxy started at 127.0.0.1:8080
2019-04-25 13:08:15.281 [INF] LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.
```

**Comment by cmoleka** [comment]
> I faced the same issue on High Sierra (v10.13.6), NL v6.0.4
> Here is log:
> 
> 2019-04-17 17:04:50.164 [INF] LTND: Version: 0.6.0-beta commit=v0.6-beta-rc4, build=production, logging=default
> 2019-04-17 17:04:50.164 [INF] LTND: Active chain: Bitcoin (network=mainnet)
> 2019-04-17 17:04:50.173 [INF] CHDB: Checking for schema update: latest_version=8, db_version=8
> 2019-04-17 17:04:50.187 [INF] RPCS: password RPC server listening on 127.0.0.1:10009
> 2019-04-17 17:04:50.199 [INF] RPCS: password gRPC proxy started at 127.0.0.1:8080
> 2019-04-17 17:04:50.200 [INF] LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.
> 
> After a few restarts, it works as expected and starts to sync.

What do you mean, after a few restarts?

**Comment by PierreRochard** [comment]
@cmoleka I think he means closing the Node Launcher and then reopening it

**Comment by sr1919** [comment]
> @cmoleka I think he means closing the Node Launcher and then reopening it

@cmoleka yes, that's it
